kover-gradle-plugin/docs/configuring.md

```diff
@@ -64,10 +64,10 @@ koverReport {
               maxValue = 99
   
               // specify which units to measure coverage for
-              metric = kotlinx.kover.gradle.plugin.dsl.MetricType.LINE
+              coverageUnits = kotlinx.kover.gradle.plugin.dsl.MetricType.LINE
   
               // specify an aggregating function to obtain a single value that will be checked against the lower and upper boundaries
-              aggregation = kotlinx.kover.gradle.plugin.dsl.AggregationType.COVERED_PERCENTAGE
+              aggregationForGroup = kotlinx.kover.gradle.plugin.dsl.AggregationType.COVERED_PERCENTAGE
```

Comment:
Same here, although `aggregationForGroup` and `AggregationType` already similar enough (but maybe we still should rename the latter to `AggregationGroup`).

Reply:
It seems to me that we can leave it here, `aggregationForGroup` means that for each group we will use some kind of data aggregation algorithm, which is determined by its type - `AggregationType`